src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  align-self: center;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  list-style: none;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link:link {
+  text-transform: uppercase;
+  text-decoration: none;
+  color: black;
+  display: block;
+  padding: 23px 0;
+}
+
+.nav__link:visited {
+  color: black;
+}
```

Comment:
If somebody visits the link and goes back to your header, your links will be black (without padding, text-decoration and text-transform will be by default). Think: use links and visited pseudo-classes is necessary for this task?